docs/codeflow/git-persistence.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Git projects and persistence
+description: 'How persistence works in the StackBlitz editor for Git repositories'
+og_image: what-is-codeflow.png
+---
+
+# {{ $frontmatter.title }}
+
+The StackBlitz editor has some differences with respect to _persistence_ (whether and how we store your changes) between _projects_ and Git repositories.
+
+## Default mode
+
+In its default mode, the StackBlitz editor **does _not_ store your changes** when you open a Git repository (for instance, https://stackblitz.com/~/github.com/vitejs/vite). You can edit files, create new commits and push them to a remote. However, any intermediate changes that are not pushed will be lost after reloading the page. The editor shows a prominent warning to remind you when persistence is off.
+
+ This is in constrast with _projects_, which always keep your changes stored in the StackBlitz backend. The rough rationale for this difference in behavior is that, for Git-based projects, the ultimate goal is to push some new changes into a remote that already exists. This remote ultimately acts as your storage.
```

Comment:
repeated use of `ultimate`. either one could be dropped and the sentence would still work